management_os/build/hosts

```diff
@@ -0,0 +1,4 @@
+
+# Put the ip address of the control server here so the management
+# os can start communicating with it once it's booted.
+192.168.0.4         control_server
```

Comment:
Generating this file is probably much nicer than hardcoding it

Reply:
yes, I thought of that (just haven't implemented it yet), but I also wanted to discuss that because generating it probably means taking the ip address of the machine it's generated on, and putting that in the hosts file. But I think we can't just assume that we generate the initramfs on the same machine that will be the control server. How do you propose this should work? Currently the generation script doesn't take any parameters (also because forwarding parameters from makefiles isn't too ergonomic)

Reply:
Env var was my initial thought, not sure if it's the best option though